auditwheel/__main__.py

```diff
@@ -1,5 +1,5 @@
 import sys
 from .main import main
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
I'm not sure what docstyle you have applied but I don't want to switch from single quotes to double quotes.

Reply:
Sure I'll make the changes as you have said and place it in a day or two. thanks.

Reply:
I believe that was due to the use of the `black` formatter as per a suggestion in the conversation thread.
Nevertheless I will revert to as per your suggestion.